features/full_tests/native_crash_handling.feature

```diff
@@ -19,7 +17,7 @@ Feature: Native crash reporting
     And the event "device.totalMemory" is not null
     And the error payload field "events.0.device.cpuAbi" is a non-empty array
     And the error payload field "events.0.metaData.app.memoryLimit" is greater than 0
-    And the first significant stack frames match:
+    And any significant stack frames match:
```

Comment:
nit: does "significant" actually mean anything now?  i.e. is it really just any frame?

Reply:
Ignore this PR - I just realised there is a better way to solve this 😆